src/components/conference/JoinConferenceButton/JoinConferenceButton.tsx

```diff
@@ -27,26 +33,42 @@ const JoinConferenceButton = ({
   tooltipPosition = 'top',
   onInitialise,
   onSuccess,
+  onError,
   testID,
   ...rest
 }: JoinConferenceButtonProps) => {
   const { createConference, joinConference } = useConference();
+  const { setErrorsExplicitly } = useErrors();
 
   const handleJoinConference = async () => {
-    if (onInitialise) {
-      onInitialise();
-    }
+    try {
+      if (onInitialise) {
+        onInitialise();
+      }
+      const newConference = await createConference({
+        alias: meetingName,
+        params: {
+          // Keep empty conferences alive for 5 minutes to allow participants to join or rejoin
+          ttl: 60 * 5,
```

Comment:
@vincentsong @Noviny enhancement idea for a subsequent release - can `ttl` be an optional input param?

Reply:
Definitely.